components/VideoBar/VideoBar.tsx

```diff
@@ -0,0 +1,158 @@
+import styled from "styled-components";
+import css from "@styled-system/css";
+import { transition } from "components/system";
+import NextImage from "next/image";
+import Icon from "components/Icon";
+import Flex from "components/Flex";
+import Button from "components/Button";
+
+const SEC_IN_MINUTE = 60;
+const SEC_IN_HOUR = 3600;
+
+const addZero = (number: number) => String(number).padStart(2, `0`);
+
+function secondsToHms(d: number) {
+  const hour = Math.floor(d / SEC_IN_HOUR);
+  const min = Math.floor((d % SEC_IN_HOUR) / SEC_IN_MINUTE);
+  const sec = Math.floor((d % SEC_IN_HOUR) % SEC_IN_MINUTE);
+
+  const hDisplay = hour > 0 ? `${hour}:` : "";
+
+  return `${hDisplay}${addZero(min)}:${addZero(sec)}`;
+}
+
+export interface VideoBarProps {
+  previewSrc: string;
+  videoLink: string;
+  name: string;
+  videoDurationSec?: number;
```

Comment:
```suggestion
  thumbnail: string;
  href: string;
  title: string;
  duration?: number;
```

We don't need `video` prefix here, because the is only video here. Also `title` and `href` and the names that we use for such props in other components, so I renamed them, for consistency.

Reply:
fixed